docs/content/users/basics/developer-tools.md

```diff
@@ -4,27 +4,27 @@ Run `ddev describe` to see the project information and services available for yo
 
 ## Command-line Tools in the Containers
 
-Hundreds of useful developer tools are included inside the containers. Any of these tools can be used via `ddev exec` or `ddev ssh`. A short listing is:
+Hundreds of useful developer tools are included inside the containers and can be used via `ddev exec` or `ddev ssh`. Some of those include:
 
 * MySQL client (`mysql`) - Command-line interface for interacting with MySQL/MariaDB.
-* Postgresql client (`psql`) - Command-line tool for Postgresql.
+* PostgreSQL client (`psql`) - Command-line tool for PostgreSQL.
 * [Drush](http://www.drush.org) - Command-line shell and Unix scripting interface for Drupal.
-* [PHIVE](https://phar.io/) - Command line tool for "PHAR Installation and Verification Environment"
-* [WP-CLI](http://wp-cli.org/) - Command-line tools for managing WordPress installations, available both as "wp" and as "wp-cli".
-* npm and yarn (and the `ddev yarn` command will run yarn for you, `ddev help yarn`)
-* node
-* sqlite3
+* [PHIVE](https://phar.io/) - Command line tool for “PHAR Installation and Verification Environment”.
+* [WP-CLI](http://wp-cli.org/) - Command-line tools for managing WordPress installations, available both as `wp` and as `wp-cli`.
+* `npm` and `yarn` (and the `ddev yarn` command will run Yarn for you, `ddev help yarn`).
+* `node`
+* `sqlite3`
 
-These tools can be accessed for single commands using [`ddev exec <command>`](cli-usage.md#executing-commands-in-containers) or [`ddev ssh`](cli-usage.md#ssh-into-containers) for an interactive bash or sh session.
+These tools can be accessed for single commands using [`ddev exec <command>`](cli-usage.md#executing-commands-in-containers) or [`ddev ssh`](cli-usage.md#ssh-into-containers) for an interactive Bash or sh session.
```

Comment:
I’m not sure what the canonical reference is, but [this page](https://www.gnu.org/software/bash/) consistently uses “Bash” as a proper noun. I can’t find any equivalent for “sh”.

Avoiding the situation with backticks works for me, since those are still literal executables. (Though you’ll find more “Bash” updates in these docs, I think!)

Reply:
It's not *that* important. I guess the overall thing is figuring out relatively consistent usage for things, a general rule for handling them. And I know you've been working on that. But we want future contributors to docs to be successful without stress.

Reply:
As an old Unix person, my personal preference is lower-cased command names for so many things. But your strategy is working fine. And of course, referring to products with the capitalization they prefer.

Reply:
Understood! IMO this effort should encourage future contributions by offering an internal consistency that aligns with how other organizations/products identify themselves.

Once we’re done here, this can be codified in a writing style guide for anyone that wants to bother reading it.